modules/swagger-codegen/src/main/resources/JavaPlayFramework/newApiController.mustache

```diff
@@ -215,21 +226,50 @@ public class {{classname}}Controller extends Controller {
             SwaggerUtils.validate(obj);
             {{/returnContainer}}
         }
+        {{/supportAsync}}
+        {{#supportAsync}}
```

Comment:
@wing328  I helped @ignaciomolina  with his PR by suggesting this solution. To have good tabulation, we need to duplicate stuff in the mustache file (look here for the bean validation).
Any idea to do it more cleanly? Otherwise, this PR looks good to me!

Reply:
@JFCote agreed with you that some files or code (refactored into mustache partial) can be reused between different generators. I've some ideas but would require moving files around (which is not an elegant solution).

Reply:
@wing328 I'm not talking about moving stuff between generators, I'm talking about duplicate stuff in this mustache file. The line that is highlighted. There is the version with supportAsync without tabulation and the version without supportAsync that doesn't have any tabulation. The code is duplicated.

I was wondering if there was a better way to do that?

Reply:
@JFCote ah ok. Just went through the mustache file you provided.

What about creating another file `newApiControllerAsync.mustache` instead of using `{{#supportAsync}} .. {{/supportAsync}}` in `newApiController.mustache`? The drawback is that we must remember applying the same enhancement/bug fixes to both files.

Reply:
That's even worse than duplicate a chunk of code xD
Maybe the best solution was the first, to just add `{{#supportAsync}}    {{/supportAsync}}` in front each line :sweat: